### PR TITLE
delete unnecessary file

### PR DESCRIPTION
**Pull request by violet-ewgd**
I have deleted  the unnecessary file vitest.config.ts.timestamp-1681276750671.mjs

### CODE REVIEW

**Comment by violet-ewgd** [comment]
For me ，this is a college assignment, can you please reply quickly? Thanks.

**Comment by Atralupus** [comment]
Related to https://github.com/planetarium/libplanet/issues/3979

**Comment by violet-ewgd** [comment]
6 workflows awaiting approval
 
6 workflows are waiting for maintainer to approve，right？ there is not failed test ，right？

**Review by riemannulus — APPROVED** [review]

**Pull request merged**
